source/preface_third.md

```diff
@@ -1,10 +1,250 @@
 # Preface to the third edition {-}
 
+One of the pleasures of writing a new edition of someone else's book is that we
+get some freedom to say good things about the earlier editions.  We get to
+praise a version of our own book.   We will do that, in the next section.  In
+the rest of the preface, we will say what we have changed, and why, and say
+a little about where this book sits in your learning and teaching about data
+analysis and statistics.
+
+## What Simon saw
+
+The book in your hands, or on your screen, is the third edition of a book
+originally called "Resampling: the new statistics", by Julian Lincoln Simon
+[-@simon1992resampling]. You will see some history of the book in Simon's
+original preface, that follows this one.  Please do read that preface; it is an
+excellent overview of what Simon was trying to do, and why.  What Simon could
+not know, when he wrote that preface, was that the method he had discovered
+would become the foundation of a revolutionary change in the teaching of data
+analysis, that we now call "data science".
+
+Simon published the first edition in 1992, but the ideas behind the book
+started much earlier, when he noticed a major problem that is familiar to
+almost every teacher of research methods - his students had a very shallow
+understanding of statistics.  The typical response to this typical insight is
+the "cookbook" method.  We try to minimize the damage by teaching recipes for
+common situations. We apply warning labels to the recipes, in the hope
+that our students will later do the right tests, even if they have little
+understanding of what those tests mean.  Of course, this is a miserable way to
+learn, and a miserable way to teach.  Our better students see that we are not
+teaching for understanding, but rehearsal, and they suffer the most.  As you
+will soon see, this book is for the teacher as well as the student.  It may be
```

Comment:
Readers may be confused here whether you mean suffer through the new or the old method (just requires a minor rephrasing).

Reply:
Done

Reply:
Check rephrasing for clarity?